apollo-compiler/src/main/kotlin/com/apollographql/apollo/compiler/OperationTypeSpecBuilder.kt

```diff
@@ -1,17 +1,29 @@
 package com.apollographql.apollo.compiler
 
-import com.apollographql.apollo.compiler.ir.*
 import com.apollographql.apollo.api.OperationName
 import com.apollographql.apollo.api.ResponseFieldMapper
+import com.apollographql.apollo.compiler.ir.*
 import com.squareup.javapoet.*
 import javax.lang.model.element.Modifier
 
 class OperationTypeSpecBuilder(
     val operation: Operation,
-    val fragments: List<Fragment>
+    val fragments: List<Fragment>,
+    useSemanticNaming: Boolean
 ) : CodeGenerator {
-  private val OPERATION_TYPE_NAME = operation.operationName.capitalize()
-  private val DATA_VAR_TYPE = ClassName.get("", "$OPERATION_TYPE_NAME.Data")
+  private val OPERATION_TYPE_NAME: String
```

Comment:
nit: not sure why we make these names to be capitalized initially, should be lower camel case I guess